src/shared/Fieldset.tsx

```diff
@@ -172,6 +172,11 @@ export const Fieldset = memo(
                     ))}
                 </div>
                 {state !== "default" && (
+                    <div
+                        className="fr-messages-group"
+                        id="radio-rich-error-messages"
```

Comment:
Never use hard codded id's, alwas resort to `useId` 
Here I think ther's no need for an id.